projects/ibmswtpm2/fuzzer.cc

```diff
@@ -0,0 +1,78 @@
+#include "TpmBuildSwitches.h"
```

Comment:
Let's upstream this target at some point (as described in our docs on [ideal integrations](https://google.github.io/oss-fuzz/advanced-topics/ideal-integration/))

Reply:
Sounds good!